GrandCentralBoard/Widgets/Bonus/BonusWidget.swift

```diff
@@ -21,7 +29,9 @@ final class BonusWidget: WidgetControlling {
         self.bubbleResizeDuration = bubbleResizeDuration
         self.numberOfBubbles = numberOfBubbles
 
-        let viewModel = WidgetTemplateViewModel(title: "BONUSLY", subtitle: "LAST \(numberOfBubbles) PEOPLE", contentView: widgetView)
+        let viewModel = WidgetTemplateViewModel(title: widgetTitle,
+                                                subtitle: "Last \(numberOfBubbles) people".localized.uppercaseString,
```

Comment:
it will not work with localization. It should be `String(format: "Last %d people".localized, numberOfBubbles).uppercaseString` .